frontends/web/src/api/account.ts

```diff
@@ -18,6 +18,7 @@ import { apiGet, apiPost } from '@/utils/request';
 import type { ChartData } from '@/routes/account/summary/chart';
 import type { TDetailStatus } from './bitsurance';
 import type { SuccessResponse } from './response';
+import { Slip24 } from 'request-address/dist/slip24';
```

Comment:
The /dist/ looks very weird :joy: Surprised this would even work. @davidknezic could you also export the Slip24 type in https://github.com/pocketbitcoin/request-address/blob/main/src/index.ts?

Reply:
lol, didn't notice it :sweat_smile:

Reply:
Just published `v0.0.9` which includes an export of the `Slip24` type.